yivi-docs/blog/2025-7-03-yivi-portal-announcement/index.md

```diff
@@ -0,0 +1,63 @@
+ As a verifier (referred to as relying party to adhere with the most recently used terminologies) you can register as an organization. You will then be able to set up your relying party. In doing so, you will need to specify which attributes (e.g. age, phone number) you will be requesting. You can add a few of these. You will also need to provide some context explaining why the combination of attributes is needed. For example, an online liquor shop can add age, home and email address with the context of fulfilling orders, sending an email confirmation and posting the package. This helps us ensure legitimacy of these requests and protecting our user's privacy from sharing unnecessary information. You will then be given a DNS challenge, with which we verify your domain ownership. 
+ When accepted, we will get started with publishing your relying party.
+Another feature now coppled with the portal is the <a>new attribute index</a> service which was previously hosted seperately <a href="https://attribute-index.yivi.app/">here</a>. You can also issue demo credentials as you did previously. This can help our consumers find all they need to know about organizations and credentials in one place.
+Starting **4th of July$** you can register your organizations or request maintainer rights for your exisitng organization.
+
+
+
+
+## Trusted Verifier 
+
+Trusted Verifier is more than anything, rooted in enhancing user privacy through UX. 
+This program is also known as pretty verifier program has been in the works as a way to further help our 
+users stay safe and make informed decisions regarding sharing their information with Yivi. Yivi makes it easy for 
+any party to set up a verification service and ask for user's data. The decision to share this information then fully relies on the user, who may be uninformed about the source of this request. The goal is to guide the user to consent only when they are sure about it. Yivi already does this to some extent, with the help of the <a href="https://github.com/privacybydesign/pbdf-requestors"> requestors scheme </a>.
+Parties have already been able to make themselves known to Yivi, by adding their branding and hostname, which would then be manually verified by our team. The app would read this and present
+user with the user friendly branding (name and logo). Otherwise users would be encountered with the raw url starting the session, which to most users 
+will seem odd and unsafe. 
+The new initiative plans to take this a step further, making it an easier and more instantanious decision for the user that what seems trusted party to share information
+with and what is not. This is done through the use of colors. Green for trusted and red for untrusted. The new update will come out on **1st of September**.
+
+
+:::warning
+**4th of July**
+
+Maintainers from organizations can register their organization in the Yivi portal
```

Comment:
Maybe good to mention why existing verifiers should also register on the portal?